File: TransCoder_tokenized_test_set_functions/python/DIRECTION_LAST_SQUARE_BLOCK.py
def direction ( R, C ) :
    if ( R != C and R % 2 == 0 and C % 2 != 0 and R < C ) :
        print ( "Left" )
        return
    if ( R != C and R % 2 == 0 and C % 2 == 0 and R > C ) :
        print ( "Up" )
        return
    if R == C and R % 2 != 0 and C % 2 != 0 :
        print ( "Right" )
        return
    if R == C and R % 2 == 0 and C % 2 == 0 :
        print ( "Left" )
        return
    if ( R != C and R % 2 != 0 and C % 2 != 0 and R < C ) :
        print ( "Right" )
        return
    if ( R != C and R % 2 != 0 and C % 2 != 0 and R > C ) :
        print ( "Down" )
        return
    if ( R != C and R % 2 == 0 and C % 2 != 0 and R < C ) :
        print ( "Left" )
        return
    if ( R != C and R % 2 == 0 and C % 2 == 0 and R > C ) :
        print ( "Up" )
        return
    if ( R != C and R % 2 != 0 and C % 2 != 0 and R > C ) :
        print ( "Down" )
        return
    if ( R != C and R % 2 != 0 and C % 2 != 0 and R < C ) :
        print ( "Right" )
        return
    

File: TransCoder_tokenized_test_set_functions/java/DIRECTION_LAST_SQUARE_BLOCK.java
static void direction ( int R, int C ) {
  if ( R != C && R % 2 == 0 && C % 2 != 0 && R < C ) {
    System . out . println ( "Left" );
    return;
  }
  if ( R != C && R % 2 != 0 && C % 2 == 0 && R > C ) {
    System . out . println ( "Up" );
    return;
  }
  if ( R == C && R % 2 != 0 && C % 2 != 0 ) {
    System . out . println ( "Right" );
    return;
  }
  if ( R == C && R % 2 == 0 && C % 2 == 0 ) {
    System . out . println ( "Left" );
    return;
  }
  if ( R != C && R % 2 != 0 && C % 2 != 0 && R < C ) {
    System . out . println ( "Right" );
    return;
  }
  if ( R != C && R % 2 != 0 && C % 2 != 0 && R > C ) {
    System . out . println ( "Down" );
    return;
  }
  if ( R != C && R % 2 == 0 && C % 2 == 0 && R < C ) {
    System . out . println ( "Left" );
    return;
  }
  if ( R != C && R % 2 == 0 && C % 2 == 0 && R > C ) {
    System . out . println ( "Up" );
    return;
  }
  if ( R != C && R % 2 == 0 && C % 2 != 0 && R > C ) {
    System . out . println ( "Down" );
    return;
  }
  if ( R != C && R % 2 != 0 && C % 2 == 0 && R < C ) {
    System . out . println ( "Right" );
    return;
  }
}